### DISCUSSION TITLE
Performance

### DISCUSSION THREAD

**Discussion by guraltsev** [hakimel/reveal.js#3335] [Q&A]
Hello everyone,

I am making my first reveal.js presentation. It has a lot of math (mathjax3), some animations and pictures (png around 4/page superimposed using ```r-stack```) and a lot of fragments (10 per page on average). The presentation is VERY slow. often, transitioning from one slide to the next (after hitting the arrow key) takes around 3 second in which the whole presentation is unresponsive. I have no idea how to debug this. It seems I am doing something wrong here. Does anyone have any insight? Thanks!

**Comment by Martinomagnifico** [comment]
Hi Gennady, could you host your presentation or a part of it somewhere?

**Reply by guraltsev** [reply]
Dear Martino,

I have just added you as a collaborator on a private repo. This was a prototype for a job talk (It is incomplete because I switched technologies for now because of these performance issues). However, I was really impressed with the framework so it would be great to figure this out. I wrote a slightly hacky theme to emulate LaTeX beamer. I think with some optimization this could lead to a wide adoption by many in the mathematics community. Many people lament the lack of animation support in PDF - based slides. 

(Please do not circulate publicly yet; I can add other developers if necessary).
